websites/R/Radiko/dist/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "Munn",
+    "id": "379616881272946688"
+  },
+  "contributors": [
+    {
+      "name": "Krauchelli",
+      "id": "229334514084741120"
+    }
+  ],
+  "service": "Radiko",
+  "description": {
+    "en": "Japanese's radio broadcast platform."
+  },
+  "url": "radiko.jp",
+  "regExp": "(radiko[.]jp)[/]",
```

Comment:
Is there a good reason for this? Otherwise I think you could get away with just the URL.

Reply:
Nothing, i just see some example from another presence. Delete it soon.